test/node.js

```diff
@@ -601,10 +605,17 @@ node.initApplication = function (cb, initScope) {
 			}]
 		}, function (err, scope) {
 			// Overwrite onBlockchainReady function to prevent automatic forging
-			scope.modules.delegates.onBlockchainReady = function () {};
+			scope.modules.delegates.onBlockchainReady = function () {
+				// Wait for genesis block's transactions to be applied into mem_accounts
+				if (!initScope.noWaitForGenesisBlock) {
```

Comment:
I also like `waitForGenesisBlock` - that can be add to options with default `true`

Reply:
Agree, `waitForGenesisBlock` is better.

Reply:
ok, hope it's the final version of requesting name change of this variable :)